app/src/main/AndroidManifest.xml

```diff
@@ -13,6 +13,7 @@
         android:name=".App"
         android:autoRevokePermissions="allowed"
         android:hardwareAccelerated="true"
+        android:enableOnBackInvokedCallback="true"
```

Comment:
I don't think we should proceed to merge this PR until the issue is completely solved. Thanks for bringing it up but if you could help fix this entire issue, that would be great.

Reply:
Okay, Thanks for your response. I thought so too. We can make the app more user-friendly by handling BackPress Events. But I want a little guidance from you on this matter, Like where we can apply it. If possible, Then I definitely want to work on this issue. 😇

Reply:
Thank you. How can I join?

Reply:
@akshaaatt how can I address you on this channel?